editor/src/components/editor/notification-bar.tsx

```diff
@@ -25,14 +25,24 @@ export const EditorOfflineBar = betterReactMemo('EditorOfflineBar', () => {
 
 export const LoginStatusBar = betterReactMemo('LoginStatusBar', () => {
   const loginState = useEditorState((store) => store.userState.loginState, 'LoginStatusBar')
+  const saveError = useEditorState(
+    (store) => store.editor.saveError,
+    'EditorComponentInner saveError',
+  )
 
   const onClickLoginNewTab = React.useCallback(() => {
     window.open(auth0Url('auto-close'), '_blank')
   }, [])
 
+  if (saveError) {
+    return <EditorOfflineBar />
+  }
```

Comment:
Is there a situation where being logged out would trigger the save error and so show this rather than the logged out bar?

Reply:
nope, because the save logic uses isLoggedIn to decide to attempt an online save or go for a local cached save:

https://github.com/concrete-utopia/utopia/blob/796e62c4a43b737eac464ea25c13d5093c12a31a/editor/src/components/editor/store/dispatch.tsx#L663-L667